context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+This is for artists who know how to program, or programmers who know how to do art. Or,perhaps, for programmers who are bored.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to add simple shapes, including TITLES - words made of squares -  since these things are built in. It's also moderately easy to recursively draw a shape, as this is also a built-in function.
+
+Plus, the 'rule's make it very easy to randomize compared to other languages.
 
 ## What is hard to do in this language? Why is it hard?
 
+It would be difficult to generate a complex image with many strokes, such as a self-portrait, or a very complex, realistic landscape, as this language generates images that are built off of squares, circles, and triangles. It would also be difficult to do anything that isn't built in, like add a conditional statement that wasn't based on randomness or a loop that didn't run until the shapes are too small to be seen.
```

Comment:
I wonder if there is a project here... to process an image and express it in ContextFree language